src/components/ProjectsMenu.vue

```diff
@@ -6,14 +6,14 @@
   <div id="menu-projects" class="container">
     <div class="row row-equal">
       <!-- item -->
-      <div v-for="menuitem in state.menuitems"  :key="menuitem.title" @click="trigger(menuitem)" class="col-xs-12 col-sm-4 project-menu">
+      <div v-for="(menuitem, index) in state.menuitems"  :key="menuitem.title" @click="trigger(menuitem)" class="col-xs-12 col-sm-4 project-menu">
         <div class="project-menu-item-image">
           <img :src="logoSrc(menuitem.thumbnail)" class="img-responsive">
         </div>
         <div class="project-menu-item-content">
           <div class="project-menu-item-text">
             <h4 class="project-menu-item-title">{{ menuitem.title }}</h4>
-            <div v-html="menuitem.description"></div>
+            <div :ref="`description_${index}`" class="project-menu-too-long-description" @mouseover="showFullDescription(index)" @mouseout="hideFullDescription(index)" v-html="menuitem.description"></div>
```

Comment:
@volterra79 Beware that "hover" elements have poor accessibility scores, even on mobile they are not so easy to use ...

Reply:
@Raruto ok